src/GPLikelihoods.jl

```diff
@@ -12,12 +12,16 @@ export BernoulliLikelihood,
     CategoricalLikelihood,
     GaussianLikelihood, 
     HeteroscedasticGaussianLikelihood, 
-    PoissonLikelihood
+    PoissonLikelihood,
+    ExponentialLikelihood,
+    GammaLikelihood
 
 # Likelihoods
 include("likelihoods/bernoulli.jl")
 include("likelihoods/categorical.jl")
 include("likelihoods/gaussian.jl")
 include("likelihoods/poisson.jl")
+include("likelihoods/gamma.jl")
+include("likelihoods/exponential.jl")
```

Comment:
Just realised we're doing these wrong (they won't work on Windows).
Could you replace these and the above with the appropriate `joinpath`, i.e.
```suggestion
include(joinpath("likelihoods", "gamma.jl"))
include(joinpath("likelihoods", "exponential.jl"))
```

Reply:
Scratch that -- our CI is passing on Windows so apparently it works fine on Windows. I'm confused. Either way, `joinpath` is probably more idiomatic.